components/tidb_query/src/rpn_expr/mod.rs

```diff
@@ -119,6 +119,14 @@ fn divide_mapper(lhs_is_unsigned: bool, rhs_is_unsigned: bool) -> RpnFnMeta {
     }
 }
 
+fn truncate_int_mapper(lhs_is_unsigned: bool, rhs_is_unsigned: bool) -> RpnFnMeta {
+    match (lhs_is_unsigned, rhs_is_unsigned) {
+        (false, false) => truncate_int_with_int_fn_meta(),
+        (true, false) => truncate_uint_with_int_fn_meta(),
```

Comment:
Since there is `TruncateUint` to truncate uint, should we remove the part in the `TruncateInt` where `lhs` is `u64`?